docs/CLI-demo.yml

```diff
@@ -0,0 +1,374 @@
+# The configurations that used for the recording, feel free to edit them
```

Comment:
I had no idea this even existed. This is super cool!

Reply:
Me neither, I think it's an awesome tool!